main.cpp

```diff
@@ -214,7 +241,7 @@ int main(void)
         } else if (in_char == 'r') {
             (void) fcc_storage_delete(); // When 'r' is pressed, erase storage and reboot the board.
             printf("Storage erased, rebooting the device.\n\n");
-            wait(1);
+            thread_sleep_for(1000);
```

Comment:
Is there some special reason for changing the `wait` out for the `thread_sleep_for`?

Reply:
i saw a deprecation notice in mbed studio for wait
```
MBED_DEPRECATED_SINCE("mbed-os-5.14",
                      "'wait' is deprecated in favor of explicit sleep functions. To sleep, 'wait' should be replaced by "
                      "'ThisThread::sleep_for' (C++) or 'thread_sleep_for' (C). If you wish to wait (without sleeping), call "
                      "'wait_us'. 'wait_us' is safe to call from ISR context.")
```

Reply:
That is definitely a good reason!-)